Notes

FrankieChik opened the pull request:
I make a new pull request here. My answers can be seen in 03Homework-answers.ipynb and Homework2(This_is_my_answers).ipynb. I do not know why I could not pull the original files from my computer so I created new files.

FrankieChik committed:
Create The very basics.ipynb

Commit:
HomeworkBasic1.ipynb

FrankieChik committed:
Merge pull request #1 from FrankieChik/patch-01

HomeworkBasic1.ipynb

FrankieChik committed:
Merge pull request #2 from FrankieChik/patch-01

HomeworkBasic1.ipynb

FrankieChik committed:
Merge pull request #3 from FrankieChik/myNew2Branch

Merge pull request #2 from FrankieChik/patch-01

jdamerow committed:
added second homework

jdamerow committed:
Added homework for week 5

jdamerow committed:
renamed first homework file

FrankieChik committed:
Merge pull request #5 from diging-training/master

Homework2

FrankieChik committed:
Merge pull request #6 from FrankieChik/patch-1

Create The very basics.ipynb

Commit:
Homework2.ipynb

Commit:
Homework2(This_is_my_answer).ipynb

Commit:
Homework2Finalanswers.ipynb

jdamerow committed:
homework assignment 6

Commit:
03Homeworkanswers.ipynb

FrankieChik committed:
Merge pull request #11 from diging-training/master

homework assignment 6

jdamerow committed:
Added homework for week 7

Commit:
Homeowrk4.ipynb

FrankieChik committed:
Merge pull request #14 from diging-training/master

Added homework for week 7

jdamerow committed:
added homework for week 8

FrankieChik committed:
Create README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

Commit:
Homework-05.ipynb

FrankieChik committed:
Merge pull request #16 from diging-training/master

added homework for week 8

FrankieChik committed:
Create index.md

FrankieChik committed:
Create secondfile

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update secondfile

Commit:
03Homework-answers.ipynb

Commit:
Homework2(This_is_my_answers).ipynb

jdamerow commented:
@FrankieChik there don't seem to be any solutions for the latest homework (Homework-06-documentation-tests), or am I missing something?

FrankieChik commented:
Hi Julia,
Did you mean homework 6? I am still working on it. Isn’t it due tonight? Thanks.
Best,
Frankie

> Julia Damerow <notifications@github.com> 於 2020年3月24日 下午1:25 寫道：
> 
> 
> @FrankieChik <https://github.com/FrankieChik> there don't seem to be any solutions for the latest homework (Homework-06-documentation-tests), or am I missing something?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/FrankieChik/python-for-the-humanities-homework/pull/17#issuecomment-603486050>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AOJI6IRP24EL4AMB7BC272DRJEJK3ANCNFSM4LS5WNDA>.
>

jdamerow commented:
yes, that's what I mean and ok, I misunderstood then. I thought the pull request was for homework 6. and yes, it's due tonight.

FrankieChik commented:
Oh, my bad. It is only for homework 2 and 3. Thanks. Will pull the latest homework tonight.

> Julia Damerow <notifications@github.com> 於 2020年3月24日 下午1:25 寫道：
> 
> 
> @FrankieChik <https://github.com/FrankieChik> there don't seem to be any solutions for the latest homework (Homework-06-documentation-tests), or am I missing something?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/FrankieChik/python-for-the-humanities-homework/pull/17#issuecomment-603486050>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AOJI6IRP24EL4AMB7BC272DRJEJK3ANCNFSM4LS5WNDA>.
>

Commit:
Homework-06-documentation-testsanswers.ipynb

FrankieChik committed:
Create project.py

FrankieChik committed:
Update project.py

FrankieChik committed:
Create project_code_04082020.py

Hi Julia and Ken,
I went over the program historians and other online topic modeling resources these two days. I am finally able to tokenize and clean the texts I am now working on. I ran the code once on Pycharm but only it came out with some numbers like [(0, 24), (1, 21), (2, 6), (3, 3), (4, 3), (5, 3), (6, 3), (7, 3), (8, 3), (9, 3), (10, 3), (11, 3), (12, 3).......
I think I may need to process topic modeling this week but may I know if I am on the right track? Thanks.
